framework/source/class/qx/event/handler/GestureCore.js

```diff
@@ -753,22 +765,6 @@ qx.Bootstrap.define("qx.event.handler.GestureCore", {
     },
 
     /**
-     * Checks if the distance between the x/y coordinates of touchstart/mousedown and touchmove/mousemove event
-     * exceeds TAP_MAX_DISTANCE and returns the result.
-     *
-     * @param event {Event} The event from the browser.
-     * @return {Boolean} true if distance is below TAP_MAX_DISTANCE.
-     */
-    isBelowTapMaxDistance: function(event) {
-      var deltaCoordinates = this._calcDelta(event);
-      var clazz = qx.event.handler.GestureCore;
-
-      return (Math.abs(deltaCoordinates.x) <= clazz.TAP_MAX_DISTANCE &&
-              Math.abs(deltaCoordinates.y) <= clazz.TAP_MAX_DISTANCE);
-    },
-
```

Comment:
You are right. But the implemention can not work. TAP_MAX_DISTANCE returns an object  )-: